Makefile

```
@@ -0,0 +1,12 @@
+default: help
+
+.PHONY : help
+help:  ## Show this help
+	@echo "$(BOLD)FUN AWS Makefile$(RESET)"
+	@echo "Please use 'make $(BOLD)target$(RESET)' where $(BOLD)target$(RESET) is one of:"
+	@grep -h ':\s\+##' Makefile | column -tn -s# | awk -F ":" '{ print "  $(BOLD)" $$1 "$(RESET)" $$2 }'
+
+.PHONY: build
+build: ## Build the container
+build:
+	docker build -t vf2m .
```

Maybe explicitly add the latest tag here:

```suggestion
	docker build -t vf2m:latest .
```